core/src/dominio/Professor.java

```diff
@@ -230,7 +230,7 @@ public String getUsername() {
         return ausencias;
     }
 
-    private void Avaliar(Aula aulaPedida, Interval primeiroDia, String motivo, List<Ausencia> ausencias) {
+    private void avaliar(Aula aulaPedida, Interval primeiroDia, String motivo, List<Ausencia> ausencias) {

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 80 characters (found 106).](https://www.codacy.com/app/calebepb/substituicao/file/2734604389/issues/source?bid=3308597&fileBranchId=3351368#l233)